package.json

```diff
@@ -1,35 +1,49 @@
 {
   "name": "react-native-responsive-layout",
-  "version": "0.4.0",
+  "version": "1.0.0",
   "description": "An easy way to implement responsive layouts.",
-  "main": "index.js",
-  "scripts": {
-    "test": "jest"
-  },
-  "peerDependencies": {
-    "react-native": ">=0.42.0",
-    "react": ">=15.0.0"
-  },
   "keywords": [
     "react-native",
     "grid",
     "layout",
     "responsive",
     "bootstrap"
   ],
-  "author": "Axilis",
+  "author": "axilis",
+  "contributors": [
+    "srolija",
+    "MislavMandaric",
+    "pero5ar"
```

Comment:
Here is ivanvarga missing, he isn't on any commit, but he was responsible to get this grid started in the first place.

Reply:
You're right, I only looked through commit stats. I'll add him. :+1: